src/compression/test.rs

```diff
@@ -0,0 +1,127 @@
+// Tests OP_COMPRESSED.  To actually test compression you need to look at
```

Comment:
Are the tests in this module actually being run in CI? I believe we'd need to update our evergreen config to enable the compression feature flags, from looking at the evergreen logs it doesn't look like these ones show up in the test output.

Reply:
No they aren't being run right now.  They only run if the feature flags are set.  I was going to update the evergreen configs in a separate PR.